src/generic_search.cpp

```diff
@@ -28,35 +28,19 @@
 #include <miopen/generic_search_controls.hpp>
 
 #include <cstddef>
-#include <limits>
 #include <chrono>
 
 namespace miopen {
 namespace solver {
 
-std::size_t GetTuningIterationsMax()
-{
-    return Value(MIOPEN_DEBUG_TUNING_ITERATIONS_MAX{}, std::numeric_limits<std::size_t>::max());
-}
+std::size_t GetTuningIterationsMax() { return Value(MIOPEN_DEBUG_TUNING_ITERATIONS_MAX{}); }
 
 std::chrono::milliseconds GetTuningTimeMax()
 {
-    static const auto fallback =
-        std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::hours{2});
-    static const auto res =
-        std::chrono::milliseconds{Value(MIOPEN_TUNING_TIME_MS_MAX{}, fallback.count())};
-    return res;
+    return std::chrono::milliseconds{Value(MIOPEN_TUNING_TIME_MS_MAX{})};
```

Comment:
[notice] This would negatively affect performance. However, the overall effects is negligible.